styles/style.css

```diff
@@ -0,0 +1,448 @@
+.body {
+  font-family: 'Gotham Pro', sans-serif;
+  margin: unset;
+  font-size: 0.75em;
+}
+.body > * {padding: 0 calc(50vw - 8.75rem)}
+
+.header__title, .product__header, .product__price, .info__header, .stories__header, .story__header {font-size: 1.666em;}
+.header__title {line-height: .95em;}
+
+.header__arrow {
+  font-size: 3.3em;
+  line-height: .22em;
+}
+
+.product__name, .story__time, .footer {font-size: 1.25em;}
+
+.info__text, .story__text {font-size: 1.166em}
+
+.photos__header {font-size: 2.08em;}
+
+.footer__text {font-size: 1.08em;}
+
+.header {color: #fff;}
+
+h1, h2, h3, h4 {margin: unset;}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+button {
+  font: inherit;
+  background: inherit;
+  border: unset;
+  padding: unset;
+  text-transform: inherit;
+  cursor: pointer;
+}
+label {cursor: pointer;}
+button:hover, a:hover, label:hover {
+  box-shadow: 0 12px 16px -16px red,0 17px 50px -50px red;
+  color: red;
+  cursor: pointer;
+  transition: all 1s;
+}
+
+.search__bar {-webkit-appearance: none;}
+textarea:focus, input:focus, button:focus {outline: none;}
+
+.product__radio {display: none;}
+
+figure {margin: unset;}
+
+.nav_caps, .header_caps, .product__header_caps, .stories__header_caps, .story__link_caps, .photos__header_caps { text-transform: uppercase; }
+.nav_bold, .footer__header, .product__price {
+  font-weight: bold;
+  text-transform: uppercase;
+}
+
+.nav {
+  display: grid;
+  grid: 4.5rem / auto auto;
+}
+
+.nav__items, .nav__links {
+  display: grid;
+  text-align: center;
+  grid-column: 1/-1;
+}
+.nav__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__link_col, .nav__signin_col, .nav__cart_col, .nav__lang_col {height: 10vw;}
+
+.nav__home {
+  display: flex;
+  align-items: center;
+}
+.nav__logo_mob {width: 6.5rem;}
```

Comment:
Codacy found an issue: [Expected single space before "}" of a single-line block (block-closing-brace-space-before)](https://app.codacy.com/gh/WebHeroSchool/r4/pullRequest?prid=5887702)